lib/plivo/resources/verify_session.rb

```diff
@@ -0,0 +1,136 @@
+module Plivo
+    module Resources
+        include Plivo::Utils
+        class Session < Base::Resource
+            def initialize(client, options = nil)
+                @_name = 'Session'
+                @_identifier_string = 'session_uuid'
+                super
+            end
+            def to_s
+                {
+                  api_id: @api_id,
+                  session_uuid: @session_uuid,
+                  app_uuid: @app_uuid,
+                  alias: @alias,
+                  recipient: @recipient,
+                  channel: @channel,
+                  status: @status,
+                  count: @count,
+                  requestor_ip: @requestor_ip,
+                  destination_country_iso2: @destination_country_iso2,
+                  destination_network: @destination_network,
+                  attempt_details: @attempt_details,
+                  charges: @charges,
+                  created_at: @created_at,
+                  updated_at: @updated_at
+                }.to_s
+            end
+        end    
+
+        class SessionInterface < Base::ResourceInterface
+            def initialize(client, resource_list_json = nil)
+                @_name = 'Verify/Session'
+                @_resource_type = Session
+                @_identifier_string = 'session_uuid'
+                super
+            end
+
+            # @param [String] session_uuid
+            def get(session_uuid)
+                perform_get(session_uuid)
+            end
+
+            def create(app_uuid = nil, recipient = nil,channel = nil,url = nil, method = nil)
+                value = app_uuid
```

Comment:
if your using hash, you can rename the app_uuid to params?

Reply:
yes, removed hash params